src/components/task-step/exercise-multiple-choice.cjsx

```diff
@@ -98,13 +100,32 @@ ExerciseReview = React.createClass
   onContinue: ->
     @props.onNextStep()
 
+  tryAnother: ->
+    task_id = TaskStepStore.getTaskId(@getId())
+    TaskStepActions.getRecovery(@getId())
```

Comment:
Should probably be named `loadRecovery` so as not to be confused with a getter/setter